src/Helpers/ConfigHelper.php

```diff
@@ -65,9 +65,9 @@ public function getClientSecret()
   }
 
   /**
-   * @return int
+   * @return float
    */
-  public function getOrderReferrerValue(): int
+  public function getOrderReferrerValue(): float
```

Comment:
Is there still a reason to keep casting the value to an integer on line 83 or can that be switched to a float?

Reply:
thanks. meant to remove that.